rpcd/playbooks/pip-lockdown.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The purpose here is to allow for the environment to update/build the
+# python wheel files from the CURRENT release, as set in the rpc_release
+# variable.
+- name: Clean up pip release files from previous versions
+  hosts: hosts:all_containers
+  user: root
+  gather_facts: False
+  tasks:
+    - name: remove pip release directory
+      file:
+        path: /root/.pip/links.d/
+        state: absent
+      tags:
+        - pip-link-cleanup
+
+- name: Place pip.conf on hosts
+  hosts: hosts
```

Comment:
Yes

On Tue, 21 Jun 2016, 17:35 Ian Cordasco, <notifications@github.com> wrote:

> In rpcd/playbooks/pip-lockdown.yml
> <https://github.com/rcbops/rpc-openstack/pull/1159#discussion_r67904259>:
>
> > +# python wheel files from the CURRENT release, as set in the rpc_release
> > +# variable.
> > +- name: Clean up pip release files from previous versions
> > +  hosts: hosts:all_containers
> > +  user: root
> > +  gather_facts: False
> > +  tasks:
> > +    - name: remove pip release directory
> > +      file:
> > +        path: /root/.pip/links.d/
> > +        state: absent
> > +      tags:
> > +        - pip-link-cleanup
> > +
> > +- name: Place pip.conf on hosts
> > +  hosts: hosts
>
> We're not targeting all_containers here too because a different playbook
> (or playbooks) in OSA will handle this for us, yes?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rcbops/rpc-openstack/pull/1159/files/3afa31a130690b95ca1179dda1621638328ec575#r67904259>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAiUVk34z3c3IifFpyIYLhltKIHDfuw1ks5qOBLHgaJpZM4I1Pt5>
> .
>